HB-UNI-Sen-DIST-US.ino

```diff
@@ -20,20 +20,30 @@
 #define CONFIG_BUTTON_PIN  8
 #define LED_PIN            4
 
+#define USE_IO_AS_GND // comment out if normal GND pin is used
+
 // 1 SENSOR
 //                  SENSOR  1
 byte SENSOR_EN_PINS[]   =  {5}; //VCC Pin des Sensors
 byte SENSOR_ECHO_PINS[] =  {6};
 byte SENSOR_TRIG_PINS[] =  {14};
+byte SENSOR_GND_PINS[] = {9}; // not used if normal GND pin is used
 
 //// 2 SENSOREN
 ////                  SENSOR  1   2
 //byte SENSOR_EN_PINS[]   =  {5,  7}; //VCC Pin des Sensors
 //byte SENSOR_ECHO_PINS[] =  {6,  3};
 //byte SENSOR_TRIG_PINS[] =  {14, 9};
 
+// battery measurement
+#define INTERNAL_VOLTAGE_MEASUREMENT // comment out if universal sensor
 #define BATT_EN_PIN        15 //A1
 #define BATT_SENS_PIN      17 //A3
+#ifdef INTERNAL_VOLTAGE_MEASUREMENT
+typedef as::BatterySensor           BatteryType;
+#else
+typedef as::BatterySensorUni<BATT_EN_PIN, BATT_SENS_PIN>  BatteryType;
```

Comment:
Reference has to be kept zero
`as::BatterySensorUni<BATT_EN_PIN, BATT_SENS_PIN, 0>  BatteryType;`